web/views/layout.erb

```diff
@@ -29,5 +28,10 @@
     </div>
     <%= erb :_footer %>
     <%= erb :_poll_js %>
+    <script type="text/javascript">
```

Comment:
Could you just add a data attribute to the body element and get the locale from that?

e.g. `<body class="admin" data-locale="<%= locale %>">` and then

```javascript
var locale = $('body').data('locale');
window.addEventListener('load', function() {
  timeago().setLocale(locale);
});
```